internal/transactionsubmission/manager_test.go

```diff
@@ -132,77 +115,54 @@ func Test_SubmitterOptions_validate(t *testing.T) {
 		{
 			name: "validate monitorService",
 			submitterOptions: SubmitterOptions{
-				DatabaseDSN:          dbt.DSN,
-				MonitorService:       tssMonitor.TSSMonitorService{},
+				DBConnectionPool:     dbConnectionPool,
+				SignatureService:     mSigService,
 				HorizonURL:           "https://horizon-testnet.stellar.org",
-				NetworkPassphrase:    network.TestNetworkPassphrase,
-				PrivateKeyEncrypter:  &utils.PrivateKeyEncrypterMock{},
-				DistributionSeed:     "SBDBQFZIIZ53A7JC2X23LSQLI5RTKV5YWDRT33YXW5LRMPKRSJYXS2EW",
 				NumChannelAccounts:   1,
 				QueuePollingInterval: 10,
 				MaxBaseFee:           txnbuild.MinBaseFee,
 			},
 			wantErrContains: "monitor service cannot be nil",
 		},
+		// {
+		// 	name: "validate EventProducer",
+		// 	submitterOptions: SubmitterOptions{
+		// 		DBConnectionPool:     dbConnectionPool,
+		// 		SignatureService:     mSigService,
+		// 		HorizonURL:           "https://horizon-testnet.stellar.org",
+		// 		NumChannelAccounts:   1,
+		// 		QueuePollingInterval: 10,
+		// 		MaxBaseFee:           txnbuild.MinBaseFee,
+		// 		MonitorService:       tssMonitorService,
+		// 	},
+		// 	wantErrContains: "event producer cannot be nil",
+		// },
```

Comment:
Same here.

Reply:
I don't think so. It's still on the sdp-multitenant branch:
https://github.com/stellar/stellar-disbursement-platform-backend/blob/b5a3319740eee01472aa5e99bd5424c6ab89d77c/internal/transactionsubmission/manager_test.go#L186-L206